src/InteractiveWindow/Editor/Commands/InteractiveWindowCommands.cs

```diff
@@ -256,6 +256,12 @@ public Task<ExecutionResult> TryExecuteCommand()
 "Ctrl-A               " + InteractiveWindowResources.CtrlAHelp,
         };
 
+        private static readonly string[] s_scriptDirectives = new[]
+        {
+"r                    " + InteractiveWindowResources.RefHelp,
```

Comment:
I'd like to see:
```
#r
#load
```

(people won't implicitly know to add a "#" just because it's a "directive")